clarity_ext/domain/analyte.py

```diff
@@ -47,9 +48,14 @@ def create_from_rest_resource(resource, is_input, udf_map, container_repo):
         # TODO: sample should be put in a lazy property, and all samples in a step should be
         # loaded in one batch
         samples = [Sample.create_from_rest_resource(sample) for sample in resource.samples]
+        is_control = False
+        # TODO: This code principally belongs to the genologics layer, but 'control-type' does not exist there
+        if resource.root.find("control-type") is not None:
```

Comment:
Where *does* `control-type` come from? 

If I understand it correctly the `resource` object is from the Genologics layer that you're referencing in the comment, because if I tried to follow the function call path I seems to end up with a call to `genologics.entities.Process()` at https://github.com/Molmed/clarity-ext/blob/074944ac6848f53c79ec415c95a7ebccc41cf9bd/clarity_ext/clarity.py#L18

I just wanted to make sure that this `if` expression was sensible, but I don't know what properties `Process` returns, so I can't tell if `root` will always exist, nor can I see where `control-type` is set.